lib/utils.js

```diff
@@ -119,11 +119,23 @@ const makeFileDirectoryIfNeeded = function (filepath) {
   }
 };
 
+
+const getBorwser = async function(url, launchOptions) {
```

Comment:
In an effort to still support Node 6.4, I'd avoid using `async` functions. It turns out to be more elegant anyway.

```suggestion
const getBrowser = function(config, launchOptions) {
```